front/pages/api/w/[wId]/tags/usage/index.ts

```diff
@@ -0,0 +1,41 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+
+import { withSessionAuthenticationForWorkspace } from "@app/lib/api/auth_wrappers";
+import type { Authenticator } from "@app/lib/auth";
+import { TagResource } from "@app/lib/resources/tags_resource";
+import { apiError } from "@app/logger/withlogging";
+import type { WithAPIErrorResponse } from "@app/types";
+import type { TagTypeWithUsage } from "@app/types/tag";
+
+export type GetTagsUsageResponseBody = {
+  tags: TagTypeWithUsage[];
+};
+
+async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<WithAPIErrorResponse<GetTagsUsageResponseBody>>,
+  auth: Authenticator
+): Promise<void> {
+  const { method } = req;
+
+  switch (method) {
+    case "GET": {
+      const tagsWithUsage = await TagResource.findAllWithUsage(auth);
```

Comment:
I wonder if it would'nt be flexible to add a withUsage param on /tags as we do for /agent_configurations ?

Reply:
I though about it, I initially discard it because it's not making the same request behind, so wanted to keep things clear. Didn't know it a similar behavior was used, I have an idea in mind, I'll do an other PR to keep changes clear